sorc/build_all.sh

```diff
@@ -112,6 +112,9 @@ fi
 #------------------------------------
 # TODO: Commented out until components aligned for build
 #source ../versions/build.ver
+if [[ "${MACHINE_ID}" == "noaacloud" ]] ; then
+   source "../versions/build.${MACHINE_ID}.ver"
+fi
```

Comment:
At the moment we are allowing all components to build with their own `build.ver` files. Once all components are aligned to build with the same library versions we will uncomment line 114 and return to sourcing the global-workflow `build.ver` at build time, which overrides all build versions that the components. I'm guessing these lines were added to do a similar library version force on all of the components that are building on the cloud? Please confirm, thanks!